doc/devel/api_changes.rst

```diff
@@ -37,6 +32,27 @@ take particular care when adding new API:
   __ https://emptysqua.re/blog/api-evolution-the-right-way/#adding-parameters
 
 
+.. _color-changes:
+
+Add or change colormaps and styles
+----------------------------------
+Visual changes are considered an API break. Generally, we do not modify existing
+colormaps or styles.
+
+We put a high bar on new colormaps and styles to prevent excessively growing them.
+Decision is case by case, but in general:
+
+- new styles and colormaps should bring something new to the table, not just being
+  a slight variation of what's already available in Matplotlib.
+- colorblindness and - in case of colormaps - perceptual uniformity should have
+  been considered.
```

Comment:
```suggestion
- colorblindness should have been considered.
```

Strongly suggest we leave perceptual uniformity out of this document.  Perceptual uniformity is not a necessary characteristic of a colormap, and often not even a desirable one.  Jet is not the default colormap in most plotting software these days, yet people still purposefully use it.  And for good reasons, in my opinion.  Matplotlib should be agnostic on this.

Reply:
@story645 I'm going with this wording. Effects are gradual and I don't want to put too strong requirements in here